### DISCUSSION TITLE
2011/03/05/%E8%B1%86%E8%B1%86%E5%94%B1%E6%AD%8C

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#316] [Announcements]
# 2011/03/05/%E8%B1%86%E8%B1%86%E5%94%B1%E6%AD%8C

过完今天，豆豆正好出生三个月。豆豆自打一个月前开始练习说话，就变成了一个话匣子，逮机会就哇啦哇啦的讲个没完。前两天又教豆豆学了一个新词：“饿”，我现在每次问豆豆“俄不饿？”，他就会回答“饿”。豆豆妈对此很不满，埋怨我也不教他点好词，比如爸爸妈妈之类的。我其实很冤枉的，我发现豆豆这么小的年龄，发元音还容易些，发辅音是非常困难的。我教他带福音音节的次，他常常是抻着脖子费了老大的劲也说不出来。所以目前只能教他说一些诸如“爱”“哦”“饿”之类只有原音的词。

https://blog.qizhen.xyz/2011/03/05/%E8%B1%86%E8%B1%86%E5%94%B1%E6%AD%8C

**Comment by ruanqizhen** [comment]
之前博客下的留言：

勺子
2011年03月7日 12:40 编辑
可耐的孩子，你会发现孩子长大是一件很快的事情。勺子现在就希望孩子就这么大就好了。。。

回复
